### PR TITLE
Add impl and test for switching language of summary report

### PR DESCRIPTION
**Pull request by r-ash**
This shows how we can implement the language switching in the summary report

We just need to update the content & tidy up when Ian gets back to us with translations

### CODE REVIEW

File: inst/report/styles.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+.text {
+  padding-left: 10px;
+  padding-top: 10px;
+  color: white;
+  font-size: 35px;
+}
+
+#translate:not(#translate[lang="t_(LANG)"]) {
```
Comment (r-ash) [inline inst/report/styles.css:35]: This means that if you want to render the report from this dir outside of package context i.e. outside of calling `generate_summary_report` that this won't work. Do you ever do that? We can work around that by having a default `styles.css` and overwriting that in `generate_summary_report`

File: inst/report/summary_report.Rmd [hunk 2]
```diff
@@ -6,53 +6,40 @@ params:
...
 output:
   html_document:
+    css: styles.css
     includes:
       in_header: header.html
     anchor_sections: false
 ---
 
-``` {css, echo = FALSE}
+::: {#translate lang="en"}
```
Comment (r-ash) [inline inst/report/summary_report.Rmd:18]: Use `:::` custom blocks https://bookdown.org/yihui/rmarkdown-cookbook/custom-blocks.html for managing inclusion of text

File: inst/report/summary_report.Rmd [hunk 3]
```diff
@@ -6,53 +6,40 @@ params:
...
+if (params$lang == "fr") {
+  print_en <- FALSE
+  print_fr <- TRUE
+} else {
+  print_en <- TRUE
+  print_fr <- FALSE
 }
+```
 
+```{r include = print_en, eval = print_en}
```
Comment (r-ash) [inline inst/report/summary_report.Rmd:36]: Use `include = FALSE` to hide the code block and `eval = FALSE` to stop the code block being evaluated. i.e. to stop evaluation and hide printing both should be `FALSE`

File: R/outputs.R [hunk 4]
```diff
@@ -761,9 +761,9 @@ generate_output_summary_report <- function(report_path,
   on.exit(unlink(tmpd, recursive = TRUE))
   withr::with_dir(tmpd, {
     fs::file_copy(list.files(system_file("report"), full.names = TRUE), ".")
-
-    rmarkdown::render("summary_report.Rmd", params = list(
-                                              output_zip = output_zip_path),
+    rmarkdown::render("summary_report.Rmd",
+                      params = list(output_zip = output_zip_path,
+                                    lang = t_("LANG")),
```
Comment (rtesra) [inline R/outputs.R:766]: @r-ash Can you please check if the lang parameter works correctly?

**Review by rtesra — APPROVED** [review]

**Pull request merged**